tests/unit/FilingHistoryList.spec.ts

```diff
@@ -699,7 +699,26 @@ describe('Filing History List - redirections', () => {
     fileCorrectionItem.trigger('click')
     await flushPromises()
 
+    // verify to display the correction dialog modal
+    const fileCorrectionDialog = wrapper.find('#correction-filing-dialog')
+    expect(fileCorrectionDialog).toBeDefined()
+
+    // verify radio buttons for client and staff
+    const clientRadioBtn = wrapper.find('#correct-client-radio')
+    expect(clientRadioBtn).toBeDefined()
+
+    // click the staff radio button
+    const staffRadioBtn = wrapper.find('#correct-staff-radio')
+    expect(staffRadioBtn).toBeDefined()
+    staffRadioBtn.trigger('click')
+    await flushPromises()
+
     // verify redirection
```

Comment:
If possible, please also test the type field in the draft correction filing. (You might be able to check if the build method was called and verify its return value.)